### PR TITLE
Enhance voting functionality permissions

### PR DESCRIPTION
**Pull request by johngian**
* View voting
 * Peers should be able to have read-only access to all votings
 * Council should be able to have read-only access to `Review` votings
* List voting
 * Peers should be able to list all votings
 * Council should be able to also list `Review` votings.

### CODE REVIEW

File: remo/voting/views.py [hunk 1]
```diff
@@ -23,8 +23,16 @@ def list_votings(request):
     """List votings view."""
     user = request.user
     polls = Poll.objects.all()
-    if not user.groups.filter(name='Admin').exists():
-        polls = Poll.objects.filter(valid_groups__in=user.groups.all())
+
+    is_admin = user.groups.filter(name='Admin').exists()
+    is_peer = user.groups.filter(name='Peer').exists()
+    is_council = user.groups.filter(name='Council').exists()
```
Comment (akatsoulas) [inline remo/voting/views.py]: Let's use `Q` objects here.

File: remo/voting/views.py [hunk 2]
```diff
@@ -132,8 +140,14 @@ def edit_voting(request, slug=None):
 def view_voting(request, slug):
     user = request.user
     poll = get_object_or_404(Poll, slug=slug)
+
+    is_council = request.user.groups.filter(name='Council').exists()
+    is_peer = request.user.groups.filter(name='Peer').exists()
+    is_review_poll = poll.valid_groups.name == 'Review'
+    read_only_perms = is_peer or (is_council and is_review_poll)
```
Comment (akatsoulas) [inline remo/voting/views.py]: Maybe the previous comment applies here too.

File: remo/voting/views.py [hunk 3]
```diff
@@ -132,8 +140,14 @@ def edit_voting(request, slug=None):
...
+
     # If the user does not belong to a valid poll group
-    if not user_has_poll_permissions(user, poll):
+    if not user_has_poll_permissions(user, poll) and not read_only_perms:
```
Comment (akatsoulas) [inline remo/voting/views.py]: Probably you need to hide the `Vote` button in template too

**Comment by johngian** [comment]
@akatsoulas r?

**Comment by johngian** [comment]
Tested locally
* With non-superuser non-admin user
  * User group=Council
    * Can list/view Rep, Council, Review votings
    * Cannot list/view Peer votings
  * User group=Peer
    * Can list/view Rep, Council, Review, Peer votings

**Pull request merged**